portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -0,0 +1,58 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.*;
```

Comment:
Generally try to avoid importing everything in a package and only import the classes you need specifically.